lib/ansible/inventory/data.py

```diff
@@ -240,19 +240,20 @@ def set_variable(self, entity, varname, value):
 
     def add_child(self, group, child):
         ''' Add host or group to group '''
-
+        added = False
         if group in self.groups:
             g = self.groups[group]
             if child in self.groups:
-                g.add_child_group(self.groups[child])
+                added = g.add_child_group(self.groups[child])
```

Comment:
`add_child_group` doesn't appear to return anything, does this result in a non-falsey value?

Reply:
it returns true/false if something was added or not